common/lib/dependabot/file_updaters/base.rb

```diff
@@ -25,6 +26,21 @@ def updated_dependency_files
         raise NotImplementedError
       end
 
+      def pr_message
+        Dependabot::PullRequestCreator::MessageBuilder.new(
+          dependencies: dependencies,
+          files: updated_dependency_files,
+          credentials: credentials,
+          source: @source,
+          # TODO: from config file:
```

Comment:
Question: how could `FileUpdater::Base` locate itself in the config file here?

Loading the config file seems doable: if we tell `FileFetcher::Base` that `.github/dependabot.ya?ml` is a dependency it will just appear in the `dependency_files` argument - right?

But within that file, we need to find the `.config[]` entry with the right `package-ecosystem` and `directory` that matches what's currently running:
* Mapping `directory` is `@source.directory`, easy peasy.
* Mapping `package-ecosystem` is _gnarly_ as the string in the config file doesn't correspond to `dependabot-core` (e.g. `gomod` in the config vs `go_modules` in dependabot-core).

Is config file parsing a pipe dream? We may want to find another way of sourcing these values, then accept `commit_message_options` as an initialization argument...

Reply:
Yeah seems like a heavy lift to pull in config file parsing to get these options pulled in, could we pass them in as arguments for now when calling `pr_message(prefix: "prod", prefix_development: "dev", include: "scope")`?

Reply:
Yep - even easier if we decouple from `FileUpdater` per https://github.com/dependabot/dependabot-core/pull/3191#discussion_r582810432